### PR TITLE
Shlomi shitrit/issue2

### PR DESCRIPTION
**Pull request by ShlomiShitrit**
I still want to do some changes, but it works for the moment. let me know @nadavWeisler

### CODE REVIEW

File: src/app/components/Chat/Chat.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+                          msgSection?.messageSection.length > 0
+                              ? msgSection?.messageSection.map(
+                                    (message, index) => (
+                                        <Message
+                                            key={index}
+                                            message={message}
+                                        />
+                                    )
+                                )
+                              : []
```
Comment (nadavWeisler) [inline src/app/components/Chat/Chat.tsx:27]: Why? its shouldnt get here

File: src/app/components/Message/Message.tsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+import { styles } from "@/app/components/Message/Message.style";
+import { MessageProps } from "@/app/general/interfaces";
+
+export default function Message({ message }: MessageProps) {
+    const isBot = message.sender === "bot";
+
+    return (
+        <Box
+            sx={{
+                ...styles.container,
```
Comment (nadavWeisler) [inline src/app/components/Message/Message.tsx:11]: add it to style

File: src/app/components/Message/Message.tsx [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+                justifyContent: isBot ? "flex-start" : "flex-end",
+            }}
+        >
+            <Box
+                sx={{
+                    ...styles.box,
+                    flexDirection: isBot ? "row" : "row-reverse",
```
Comment (nadavWeisler) [inline src/app/components/Message/Message.tsx:18]: add it to style

File: src/app/components/Message/Message.tsx [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+                }}
+            >
+                <Avatar
+                    sx={{ bgcolor: isBot ? "primary.main" : "secondary.main" }}
+                >
```
Comment (nadavWeisler) [inline src/app/components/Message/Message.tsx:23]: add it to style

File: src/app/components/Message/Message.tsx (hunk 2 above)
Comment (ShlomiShitrit) [reply src/app/components/Message/Message.tsx]: Done

File: src/app/components/Message/Message.tsx (hunk 3 above)
Comment (ShlomiShitrit) [reply src/app/components/Message/Message.tsx]: Done

File: src/app/components/Message/Message.tsx (hunk 4 above)
Comment (ShlomiShitrit) [reply src/app/components/Message/Message.tsx]: Done

**Comment by ShlomiShitrit** [comment]
@nadavWeisler 
review pls

**Pull request merged**